### DISCUSSION TITLE
Is it possible to integrate Navbuddy as a bottom item?

### DISCUSSION THREAD

**Discussion by TimothyKrell** [folke/edgy.nvim#25] [Q&A] [answered]
I tried to get Navbuddy to display at the bottom of the window instead of a floating window by using `ft = 'Navbuddy'` but this only put the center pane in the Edgy window. Any ideas how / if this is possible? 

https://github.com/SmiteshP/nvim-navbuddy

**Comment by distek** [comment, marked as answer]
From what I see in their code, it's 3 separate floating windows that they draw custom borders on to make it appear as a single window. They only assign the middle buffer the `Navbuddy` filetype.
I think you'd need to request from the plugin author that all 3 of the buffers have their own filetype. Something like `NavbuddyLeft`, `Navbuddy` (to keep compatibility), and `NavbuddyRight` and use that to make a custom edgy panel.

**Reply by TimothyKrell** [reply]
Cool. I'll investigate that angle. Thanks
